addon/-private/system/store.js

```diff
@@ -1128,14 +1128,13 @@ Store = Service.extend({
     let internalModel = this._internalModelsFor(modelName).get(trueId);
 
     if (!internalModel) {
```

Comment:
Ideally we can refactor this to avoid `if (!` with an else clause (and instead flipping the blocks around to remove the `!`). Definitely fine for a follow up PR though.